kotest-core/src/commonMain/kotlin/io/kotest/core/config/Project.kt

```diff
@@ -242,6 +258,19 @@ object Project {
    }
 }
 
+/**
+ * Test naming strategies to adjust test name case.
+ *
+ * @property AS_IS For: should("Happen something") yields: should Happen something
+ * @property SENTENCE For: should("Happen something") yields: Should happen something
+ * @property LOWERCASE For: should("Happen something") yields: should happen something
+ */
+enum class TestNameCaseOptions {
```

Comment:
I think TestNameCase is a better name because there are not multiple options controlled by this flag, but that single setting.